PublicationsRouter/v2/api/API.md

```diff
@@ -585,3 +586,52 @@ If the notification content is found and authentication succeeds you will receiv
     Binary Package
 
 Note that a successful access by a Repository account user will log a successful delivery of content notification into PubRouter (used for reporting on PubRouter's ability to support REF compliance).
+
+# API for matching configuration
```

Comment:
Suggest:
`# Matching configuration API`
or
`# API for retrieving/updating matching configuration`